Applications/Games/Subnautica/Steam/script.js

```diff
@@ -15,21 +15,24 @@ var installerImplementation = {
             .author("Zemogiter")
             .applicationHomepage("https://unknownworlds.com/subnautica/")
             .wineDistribution("upstream")
-            .wineVersion("4.0-rc1")
+            .wineVersion("4.0")
```

Comment:
This is exactly what we wanted to avoid. Please use the latest stable version or latest development version.

Reply:
If we were to do that and the next stable/dev version of wine introduced some regression, we would receive a bunch of issue reports down the line that we won't be able to fix because it wouldn't be an issue on our end. This applies not just to Subnautica but any app in theory. By using specific versions of Wine we ensure the script will work not just for us but for everyone else.

Reply:
Put LATEST_STABLE_VERSION and it will stay at 4.0 for a year ^^.